bolt/bolt.py

```diff
@@ -0,0 +1,141 @@
+"""
+Bolt Class
+---
+The main class for Bolt.
+
+Copyright (c) 2020 classabbyamp, 0x5c
+Released under the BSD 3-Clause License.
+"""
+
+
+import traceback
+import sys
+import random
+from datetime import datetime, time
+from typing import Optional, Dict, Tuple, List, Union, Iterable, Callable
+
+import pytz
+
+import discord
+from discord.ext import commands, tasks
+
+from .utils import misc, embeds
+from .dataclasses import BoltStatus, StatusMode
+
+
+class Bolt(commands.Bot):
+    def __init__(self,
+                 command_prefix: Union[str, Iterable[str], Callable],
+                 debug_mode: bool = False,
+                 display_prefix: str = None,
+                 prefix_error_exclude: Optional[List[str]] = None,
+                 bolt_status: Optional[BoltStatus] = None,
+                 auto_reacts: Optional[Dict[int, Tuple[str]]] = None,
+                 *args, **kwargs) -> None:
+        super().__init__(command_prefix, *args, **kwargs)
+        self.display_prefix = display_prefix
+        self.debug_mode = debug_mode
+        self.prefix_error_exclude = prefix_error_exclude
+        self.bolt_status = bolt_status
+        self.auto_reacts = auto_reacts
+        if isinstance(command_prefix, str):
+            self.display_prefix = command_prefix
+        else:
+            if display_prefix is None:
+                raise ValueError("You must pass 'display_prefix' if you don't pass a string as 'command_prefix'.")
+
+    # --- Event handlers ---
+    async def on_message(self, message: discord.Message):
+        msg = message.content.lower()
+        if self.auto_reacts is not None:
+            for emoji, keywords in self.auto_reacts.items():
+                if any([keyword in msg for keyword in keywords]):
+                    try:
+                        if e := discord.utils.find(lambda x: x.id == emoji, self.emojis):
+                            await message.add_reaction(e)
+                        else:
+                            print(f"Error: Unvalid or unnaccessible emoji in config: {emoji}")

```

Comment:
just a small grammar change
```suggestion
                            print(f"Error: Invalid or inaccessible emoji in config: {emoji}")
```